docs/api/covidcast-signals/hhs.md

```diff
@@ -0,0 +1,125 @@
+| `sum_confirmed_suspected_admissions_1d` | state | 1 day | Sum of adult and pediatric confirmed and suspected COVID-19 hospital admissions occuring each day. |
+| `confirmed_admissions_7d` | county, hrr, msa, state | 7 days | Sum of adult and pediatric confirmed COVID-19 hospital admissions occuring over a span of 7 days. |
+| `sum_confirmed_suspected_admissions_7d` | county, hrr, msa, state | 7 days | Sum of adult and pediatric confirmed and suspected COVID-19 hospital admissions occuring over a span of 7 days. |
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Estimation
+
+### Statewise, daily resolution
+
+Statewise daily resolution signals use the following four columns from
+the HHS state timeseries dataset:
+
+* `previous_day_admission_[adult|pediatric]_covid_[confirmed|suspected]`
+
+The `confirmed` signal is the sum of the two `confirmed` columns: 
+
+* adult
+* pediatric
+
+The `sum_confirmed_suspected` signal is the sum of all four columns:
+
+* adult confirmed
+* adult suspected
+* pediatric confirmed
+* pediatric suspected
+
+The source data specfies that admissions occured on the previous
+day. We automatically adjust the date of each result so that
+admissions are incident on that date.
+
+### Facilitywise, 7-day resolution
+
+Facilitywise 7-day resolution signals use the following four columns
+from the HHS facilitywise dataset:
+
+* `previous_day_admission_[adult|pediatric]_covid_[confirmed|suspected]_7_day_sum`
+
+The `confirmed` signal is the sum of the two `confirmed` columns: 
+
+* adult
+* pediatric
+
+The `sum_confirmed_suspected` signal is the sum of all four columns:
+
+* adult confirmed
+* adult suspected
+* pediatric confirmed
+* pediatric suspected
+
+The source data specfies that admissions for the previous day are
+summed over a seven-day period starting with the collection date. We
+automatically adjust the date for each result so that admissions are
+incident to the seven day period starting with that date.
+
+The source data uses an unusual week definition (originally
+Friday-Thursday; Thursday-Wednesday with our adjustment). Because this
+definition does not match that used by epiweeks, this dataset uses
+`time_type=day`, however data will only ever be present for dates that
+are the start of a seven-day collection period (ie Thursdays).
+
+Facilitywise data is aggregated to other geographies using the FIPS
+code of the facility, if available, and the ZIP5 code of the facility
+otherwise.
+
+## Limitations
+
+TBD
+
+## Lag and Backfill
+
+TBD
+
+## Source and Licensing
+
+This indicator mirrors and lightly aggregates data originally
+published by the U.S. Department of Health & Human Services under an
+[Open Data Commons Open Database License
+(ODbL)](https://opendatacommons.org/licenses/odbl/1-0/). The ODbL
+permits sharing, transformation, and redistribution of data or derived
```

Comment:
Restructure if it doesn't change meaning, "The ODbL permits sharing, transformation, and redistribution of data or derived works so long as all public uses are attributed and distributed under the ODbL."

Reply:
yeah i was worried that phrasing would attach "attributed" to "the ODbL". maybe "distributed under the ODbL and properly attributed"?

Reply:
That would work. It reads a lot better that way.